packages/contractkit/src/contract-utils-v2.ts

```diff
@@ -1,12 +1,12 @@
-import { Exchange } from '@celo/contractkit/types/Exchange'
-import { GasPriceMinimum as GasPriceMinimumType } from '@celo/contractkit/types/GasPriceMinimum'
-import { GoldToken as GoldTokenType } from '@celo/contractkit/types/GoldToken'
-import { StableToken as StableTokenType } from '@celo/contractkit/types/StableToken'
 import { CURRENCY_ENUM as Tokens } from '@celo/utils'
 import BigNumber from 'bignumber.js'
 import Web3 from 'web3'
 import { TransactionObject } from 'web3/eth/types'
 import { TransactionReceipt } from 'web3/types'
+import { Exchange } from '../types/Exchange'
```

Comment:
All the other imports from `../types` follow the pattern of `{ Foo as FooType }`. Why does this one break from that pattern?
(I see it was already like this before your change, so this is possibly out of scope)

Reply:
I agree we should make this consistent in the contractkit everywhere. I'll send a follow-on PR.